Resources/Prototypes/Entities/Effects/chemistry_effects.yml

```
@@ -238,3 +238,33 @@
+      state: resin-east
+    - map: [ "enum.EdgeLayer.North" ]
+      state: resin-north
+    - map: [ "enum.EdgeLayer.West" ]
+      state: resin-west
+  - type: SmoothEdge
+  - type: IconSmooth
+    mode: NoSprite
+  - type: FoamVisuals
+    animationTime: 0.6
+    animationState: resin-dissolve
+  - type: AtmosResinDespawn
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)